src/microbe_stage/AgentProjectile.cs

```diff
@@ -19,10 +28,22 @@ public void OnTimeOver()
 
     public override void _Ready()
     {
+        particles = GetNode<Particles>(ParticlesPath);
```

Comment:
I think the path here could be hard coded as no other paths in the agent scene are configurable, and this is pretty closely tied to the code, so I don't see the need to have a configurable path here.